src/app/static/src/js/components/runWorkflow/runWorkflow.vue

```diff
@@ -10,24 +10,39 @@
                          :submit-label="toggleFinalStepNextTo"
                          @cancel="handleCancel"
                          @complete="handleComplete"
+                         @update-run-workflow-metadata="updateRunWorkflowMetadata"
                          :disable-rename="disableRename"
                          :initial-run-workflow-metadata="runWorkflowMetadata">
         </workflow-wizard>
+        <div v-if="createdWorkflowKey" id="view-progress-link" class="text-secondary mt-2 pl-3">
+            <a @click.prevent="$emit('view-progress', createdWorkflowKey)" href="#">View workflow progress</a>
+        </div>
```

Comment:
Let me know if you thrink the link should live in another component

Reply:
Not sure what you mean by the endpoint being so fast - a real workflow would typically take many minutes or hours to run. But I don't see any reason to provide inline progress updates, given we have a page dedicated to this.

Reply:
Yeah, agreed, this just needs to provide a link to the progress page.